chunky/src/java/se/llbit/chunky/resources/Texture.java

```diff
@@ -931,7 +931,8 @@
   protected int height;
   protected int avgColor;
   private float[] avgColorLinear;
-  private float[] linear;
+  private boolean useAverageColor = false;
```

Comment:
Why is this stored per texture?

Reply:
I didn't want to change the behavior, the value value is stored depending on the value value of the option at the moment `setTexture` is called (previously by storing something different into linear, now by remembering the flag) but it is true I can make it static (in which case i need to initialize `avgColorFlat` everytime but this isn't a problem)